cpp/src/arrow/array/statistics.h

```diff
@@ -82,6 +83,11 @@ struct ARROW_EXPORT ArrayStatistics {
   /// and when set to `double`, it represents `approximate_distinct_count`.
   std::optional<CountType> distinct_count = std::nullopt;
 
+  /// \brief The maximum length in bytes among the rows of an array, may not be set
```

Comment:
The `average_byte_width` document uses "of a row" but this uses "among the rows". What is the difference of them? 
What is expected `max_byte_width` for `["a", "ab", "abc", "x"]`? I think it's `3` (`max(1, 2, 3, 1)`). Does "among the rows" means  `7` (`1 + 2 + 3 + 1`)?

Reply:
Yes.
In general, we should use the same phrasing for the same meaning to avoid confusing as much as possible.

(I'm not good at English. So I can't determine whether "of a row" or "among the rows" is better here... @lidavidm What do you think about this?)

Reply:
Either is understandable, but `of a row in the array` or `of the rows in an array` is best IMO

Reply:
Let's use it.